### DISCUSSION TITLE
Check for Updates

### DISCUSSION THREAD

**Discussion by PhilTU** [NightscoutFoundation/xDrip#2944] [General]
When does the version offered as an update get updated ? Recently it has become apparent that getting users to update to a version that will handle the new EU C5 Libre sensors is problematic because they think this means just check for updates in the three dots menu.

I do understand the channels and the meaning of the stable release etc, but when a significant issue like a new sensor tpye was addressed 6 weeks ago it would be nice if this got pushed out to users rather than having dozens of them asking for help sat with an early may release that in their eye is "fully up to date".

**Comment by Navid200** [comment]
Can you provide a link to the update you are referring to that "... handles the new EU C5 Libre ..."?

**Comment by Navid200** [comment]
A stable release comes out on average once every 6 months.

xDrip is managed 100% by volunteers.
Suggestions are always welcome.

**Reply by PhilTU** [reply]
https://github.com/NightscoutFoundation/xDrip/releases/tag/2023.05.25

![image](https://github.com/NightscoutFoundation/xDrip/assets/76240903/a5757bc5-1b68-4494-9381-7cd98124e049)

https://github.com/NightscoutFoundation/xDrip/discussions/2840 refers

**Reply by Navid200** [reply]
I do what I can to create release notes.  https://navid200.github.io/xDrip/docs/ReleaseNotes.html
The coverage may not be as accurate for Libre since I don't use Libre myself.
But, if a note is not accurate, you can tell me and I will correct it.

xDrip is used by many who use a variety of data sources.
A release that is game changing for one may be irrelevant for another.
A user can look at the release notes and decide which release they may need.

Nightscout is an open source project.  Their last stable release was on September 30, 2022.
We have had two stable releases since then already.    Looking at the big picture, I don't think we are the project that needs criticism in that respect.

It is not practical to have a stable release more often than once every 6 months.
So many changes are made in so many different data sources.  We need it all to settle before we make it a stable release.

**Reply by PhilTU** [reply]
OK, I understand and somehwat expected that.

I don't know what proportion of xDrip+ users are Libre users, perhaps it is insignificant or deemed unimportant. But a high proportion of them will need to move to the 25/5/23 version or later  ( "More Libre 2 sensor codes" in your release notes ) over the coming weeks and the current "stable release" is going to be useless for these people. 

Regular users do not look at notes, visit Github or do anything beyond ask the app to update and assume it is updated as necessary, which in this case it is not. So the "stable release" is not fit for purpose for these users. Your note is OK, but nobody's going to see it.

I use much old software myself, which is stable and does what is required, like your Nightscout example. I don't really see a parallel with having a version that doesn't work anymore for a portion of the userbase.

Perhaps the situation is exaggerated because the "stable but not useable for EU Libre 2 C5 type sensors" release came only 3 weeks before the fix for said sensors.

**Comment by Navid200** [comment]
The current Nightly has been promoted to Alpha.
So, people who are on the beta (default) update channel who are not getting the update you mentioned, can switch to the alpha channel to get it.

**Comment by PhilTU** [comment]
> The current Nightly has been promoted to Alpha. So, people who are on the beta (default) update channel who are not getting the update you mentioned, can switch to the alpha channel to get it.

Thanks for letting me know. That'll help some resolve their issue.
